source/reference/minio-mc/mc-support-profile.rst

```diff
@@ -0,0 +1,133 @@
+======================
+``mc support profile``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support profile
+
+Description
+-----------
+
+The :mc-cmd:`mc support profile` command runs a system profile for your deployment.
+The results of the profile can provide insight into the MinIO server process running on a given node.
+
+The resulting report is intended for use by MinIO Engineering.
+You can upload the report to |SUBNET|.
+Independent or third-party use of these profiles for diagnostics and remediation is done at your own risk.
+
+
+Subcommands
+-----------
+
+.. mc-cmd:: start
```

Comment:
The `start/stop` subcommands have been removed recently.

```
./mc support profile --help
NAME:
  mc support profile - generate profile data for debugging

USAGE:
  mc support profile [FLAGS] TARGET

FLAGS:
  --duration value              start profiling for the specified duration in seconds (default: 10)
  --type value                  profiler type, possible values are 'cpu', 'cpuio', 'mem', 'block', 'mutex', 'trace', 'threads' and 'goroutines' (default: "cpu,mem,block,mutex,threads,goroutines")
  --config-dir value, -C value  path to configuration folder (default: "/home/shireesh/.mc")
  --quiet, -q                   disable progress bar display
  --no-color                    disable color theme
  --json                        enable JSON lines formatted output
  --debug                       enable debug output
  --insecure                    disable SSL certificate verification
  --help, -h                    show help
  
EXAMPLES:
  1. Fetch CPU profiling only
     $ mc support profile --type cpu myminio/

  2. Fetch CPU, Memory and Block profiling concurrently
     $ mc support profile --type cpu,mem,block myminio/

  3. Fetch CPU, Memory and Block profiling concurrently for 10 minutes
     $ mc support profile --type cpu,mem,block  --duration 600 myminio/
```

Reply:
Page updated!